LayoutTests/fast/forms/range/range-stepup-stepdown-expected.txt

```diff
@@ -7,167 +7,64 @@ function arguments are (min, max, step, value, [stepCount])
 Using the default values
 PASS stepUpExplicitBounds(null, null, null, "") is "51"
 PASS stepDownExplicitBounds(null, null, null, "") is "49"
-
 Non-number arguments (stepCount)
 PASS stepUpExplicitBounds(null, null, null, "0", "0") is "0"
 PASS stepDownExplicitBounds(null, null, null, "0", "0") is "0"
 PASS stepUpExplicitBounds(null, null, null, "0", "foo") is "0"
 PASS stepDownExplicitBounds(null, null, null, "0", "foo") is "0"
 PASS stepUpExplicitBounds(null, null, null, "0", null) is "0"
 PASS stepDownExplicitBounds(null, null, null, "0", null) is "0"
-
 Normal cases
 PASS stepUpExplicitBounds(null, null, null, "0") is "1"
 PASS stepUpExplicitBounds(null, null, null, "1", 2) is "3"
 PASS stepUpExplicitBounds(null, null, null, "3", -1) is "2"
 PASS stepDownExplicitBounds("-100", null, null, "2") is "1"
 PASS stepDownExplicitBounds("-100", null, null, "1", 2) is "-1"
 PASS stepDownExplicitBounds("-100", null, null, "-1", -1) is "0"
-
-Fractional cases
-PASS stepUpExplicitBounds(-10, 10, 1, "0.1") is "1"
```

Comment:
These are below deleted tests are same as in `range-stepup-stepdown-from-renderer.html`.

Since they are duplicated - I am deleting them from here.